build/build.xml

```diff
@@ -834,8 +834,21 @@
 
   <!-- Retrieve tool -->
   <target name="untar-unzip-download" depends="untar-unzip-check" unless="${archive_file}_available">
+      <antcall target="untar-unzip-download-web" />
+      <antcall target="untar-unzip-download-local" />
+  </target>
+  <target name="untar-unzip-download-web" unless="local_sources">
```

Comment:
Is this distinction really needed?

AFAIR if you copy the needed archives in the right place before launching the build, the file is not downloaded again (this is done in the `untar-unzip-check` that is a dependency of `untar-unzip-download`).